### PR TITLE
Update the cpu check function

### PR DESCRIPTION
**Pull request by hsmj1412**
The cpu will not auto to online in fedora

Signed-off-by: Junxiang Li <junli@redhat.com>

### CODE REVIEW

**Comment by vivianQizhu** [comment]
@huangyum

File: virttest/virt_vm.py [hunk 1]
```diff
@@ -1397,6 +1397,30 @@ def get_cpu_count(self):
         finally:
             session.close()
 
+    def get_online_cpu_count(self):
```
Comment (balamuruhans) [inline virttest/virt_vm.py]: this method is exact duplicate of `get_cpu_count()`.

Comment (balamuruhans) [reply virttest/virt_vm.py]: instead we can have fix in existing method with `check_cmd` as extra argument that could default to "cpu_chk_cmd",
```python
def get_cpu_count(self, check_cmd="cpu_chk_cmd"):
    session = self.wait_for_login()
    cmd = self.params.get(check_cmd)
    try:
        out = session.cmd_output_safe(cmd)
        return int(re.search("\d+", out, re.M).group())
    finally:
        session.close()
```

Comment (hsmj1412) [reply virttest/virt_vm.py]: Thanks, a better way.

**Comment by hsmj1412** [comment]
@balamuruhans updated. Please help review

File: shared/cfg/guest-os/Windows.cfg [hunk 2]
```diff
@@ -27,6 +27,7 @@
     cdrom_cd1 = isos/windows/winutils.iso
 
     cpu_chk_cmd = echo %NUMBER_OF_PROCESSORS%
+    cpu_chk_all_cmd = echo %NUMBER_OF_PROCESSORS%
```
Comment (balamuruhans) [inline shared/cfg/guest-os/Windows.cfg]: how does this is different from `cpu_chk_cmd` for windows ?

Comment (hsmj1412) [reply shared/cfg/guest-os/Windows.cfg]: just avoid get cpu_chek_all_cmd in windows will return None

Comment (balamuruhans) [reply shared/cfg/guest-os/Windows.cfg]: in windows, it will never call `vm.get_cpu_count("cpu_chk_all_cmd")` as the check in L2364 will never succeed.

Comment (hsmj1412) [reply shared/cfg/guest-os/Windows.cfg]: Updated

**Comment by hsmj1412** [comment]
@vivianQizhu Hi, any update? Or the PR will be merged.

**Comment by vivianQizhu** [comment]
@hsmj1412 No comments, please go ahead.

**Pull request merged**